src/ship.js

```diff
@@ -0,0 +1,26 @@
+class Ship {
+	constructor(length) {
+		this.length = length;
+		this.position = [];
+		this.hp = length;
+	}
+
+	damage () {
+		this.hp -= 1
+	}
+
+	isSunk() {
+		if (this.hp === 0) {
+			return true
+		} else {
```

Comment:
If our `if` branch is already returning something, we don't need an `else` statement just to return another if the `if` branch doesn't hold true.